src/module/actor/sheet/base.ts

```diff
@@ -418,11 +418,44 @@ abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorSheet<TActo
 
         // Sell treasure item
         $html.find(".item-sell-treasure").on("click", async (event) => {
-            const itemId = $(event.currentTarget).parents(".item").attr("data-item-id") ?? "";
+            const $li = $(event.currentTarget).closest("[data-item-id]");
+            const itemId = $li.attr("data-item-id") ?? "";
             const item = this.actor.inventory.get(itemId);
-            if (item?.isOfType("treasure") && !item.isCoinage) {
-                await item.delete();
-                await this.actor.inventory.addCoins(item.assetValue);
+            const sellItem = async (): Promise<void> => {
+                if (item?.isOfType("treasure") && !item.isCoinage) {
+                    $li.slideUp(200, () => this.render(false));
+                    await item.delete();
+                    await this.actor.inventory.addCoins(item.assetValue);
+                }
+            };
+
+            if (event.ctrlKey) {
+                sellItem();
+                return;
+            }
+
+            if (item) {
+                const content = await renderTemplate("systems/pf2e/templates/actors/sell-treasure-dialog.hbs", {
```

Comment:
This is a bit too small to warrant a template. Create the element with the DOM API and get its outerHTML

Reply:
I set it now to
``content: `<p class="note">${LocalizePF2e.translations.PF2E.SellItemQuestion} '${item.name}'?</p>`,``
Is that sufficient or is there a better way?

Reply:
> I set it now to
> `` content: `<p class="note">${LocalizePF2e.translations.PF2E.SellItemQuestion} '${item.name}'?</p>`, ``
> Is that sufficient or is there a better way?

This should use game.i18n.format() with the variable included in the template, so that it can be localized correctly.

Reply:
it should be using format() since you can't be sure of the sentence structure and punctuation in each language will be the same.